tslint.json

```diff
@@ -1,5 +1,7 @@
 {
-    "extends": [
-        "@dcos/tslint-config"
-    ]
+  "extends": ["@dcos/tslint-config"],
+  "rules": {
+    "interface-name": [false],
+    "no-submodule-imports": [false]
```

Comment:
If you change this, please also remove  all comments like `// TODO: remove this disable with https://jira.mesosphere.com/browse/DCOS_OSS-3579` and all the `// tslint:disable:no-submodule-imports` and `// tslint:disable-next-line:no-submodule-imports`

Reply:
I think I did (at least in my code) 🤔 can check again 👍

Reply:
In your code yes, but I am talking about the entire codebase